zerver/migrations/0310_split_create_stream_policy.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.17 on 2021-01-14 09:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0309_userprofile_can_create_users'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='realm',
+            name='create_stream_policy',
+        ),
+        migrations.AddField(
+            model_name='realm',
+            name='create_private_stream_policy',
+            field=models.PositiveSmallIntegerField(default=1),
+        ),
+        migrations.AddField(
+            model_name='realm',
+            name='create_public_stream_policy',
+            field=models.PositiveSmallIntegerField(default=1),
+        ),
+    ]
```

Comment:
This migration needs an intermediate RunPython step, after the AddField steps and before the RemoveField step, to copy the current value into the two new fields -- otherwise, we're resetting every organization that's configured this to the default values.